### ISSUE TITLE
Look for a better base box for Vagrant

### ISSUE THREAD

**Issue by zmughal** [project-renard/devops#4]
The `debian/jessie64` box used in #3 does not use the guest additions for mounting shared folders.

```
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: No guest additions were detected on the base box for this VM! Guest
    default: additions are required for forwarded ports, shared folders, host only
    default: networking, and more. If SSH fails on this machine, please install
    default: the guest additions and repackage the box to continue.
    default:
    default: This is not an error message; everything may continue to work properly,
    default: in which case you may ignore this message.
```

The base box does _not_ have the VirtualBox guest additions installed, so syncing the shared folders is done the slow way using `rsync` rather than mounting using the `vboxsf` FS type. We should look for a better base box. I've used Ubuntu boxes with Vagrant in the past and they were setup correctly.

**Comment by zmughal** [comment]
I was looking at the Vagrant box public catalog and came across [debian/contrib-jessie64](https://atlas.hashicorp.com/debian/boxes/contrib-jessie64) which is described as

> Vanilla Debian 8 "Jessie" build with contrib vboxsf kernel module

**Comment by zmughal** [comment]
Closed by #6.
